install/kubernetes/cilium/values.yaml

```diff
@@ -86,11 +86,11 @@ rollOutCiliumPods: false
 image:
   override: ~
   repository: "quay.io/cilium/cilium"
-  tag: "v1.11.9"
+  tag: "v1.11.10"
   pullPolicy: "IfNotPresent"
   # cilium-digest
-  digest: "sha256:a732e57cb4881abe4783562bbba0045209ef85542372b44ce61584c887c49878"
-  useDigest: true
+  digest: ""
+  useDigest: false
```

Comment:
might I know why we change this to false ?

Reply:
Yes, at this point in time we cannot know the sha for the official 1.11.10 images because this PR isn't merged yet. We need to change the version number etc. and arrive on a final set of code that will be tagged, then once we tag we can build images and get the sha. So the only way to update the helm charts is _after_ this PR is merged and the release is tagged.

A subsequent step of the release process will update these to the correct new shas and set `useDigest` back to true, then we'll use that version of the helm charts to push as the official helm release.

Reply:
ah I got it now :heavy_check_mark: